### PR TITLE
DNM - Test nvidia packages

### PR DESCRIPTION
**Pull request by jaimergp**
Testing https://github.com/conda-forge/cudatoolkit-feedstock/issues/62

### CODE REVIEW

**Comment by jaimergp** [comment]
Works with a GPU!

```
(openmm76b) bash-4.2$ python -m openmm.testInstallation

OpenMM Version: 7.6
Git Revision: c74510f7f492087f24d43754e8a7d2ee1d0790ee

There are 4 Platforms available:

1 Reference - Successfully computed forces
2 CPU - Successfully computed forces
3 CUDA - Successfully computed forces
4 OpenCL - Successfully computed forces

Median difference in forces between platforms:

Reference vs. CPU: 6.30157e-06
Reference vs. CUDA: 6.72264e-06
CPU vs. CUDA: 7.40276e-07
Reference vs. OpenCL: 6.76294e-06
CPU vs. OpenCL: 8.11157e-07
CUDA vs. OpenCL: 2.69187e-07

All differences are within tolerance.
```

```
(openmm76b) bash-4.2$ python simulatePdb.py 
#"Step","Potential Energy (kJ/mole)","Temperature (K)"
1000,-143248.37343246874,284.79552492043405
2000,-140757.77968246874,299.03117943602945
3000,-140438.68593246874,299.77527740972084
4000,-139951.12343246874,299.08950333172356
5000,-140782.43593246874,299.46372864444976
6000,-140673.43593246874,304.23982936299103
7000,-140906.49843246874,306.2290171298301
8000,-140895.18593246874,299.1541129558899
9000,-141118.84218246874,298.6297685014683
10000,-140646.87343246874,302.92052745926077
```

```
(openmm76b) bash-4.2$ python benchmark.py --test=rf --seconds=10 --platform=CUDA
Platform: CUDA
Precision: single

Test: rf
Ensemble: NVT
Step Size: 4 fs
Integrated 25093 steps in 9.68731 seconds
895.206 ns/day
```

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -57,11 +62,13 @@ docker_image:                                   # [os.environ.get("BUILD_PLATFOR
   - quay.io/condaforge/linux-anvil-cuda:11.0    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
   - quay.io/condaforge/linux-anvil-cuda:11.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
   - quay.io/condaforge/linux-anvil-cuda:11.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-cuda:11.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
```
Comment (peastman) [inline recipe/conda_build_config.yaml:65]: Is this supposed to be 11.3?

**Comment by peastman** [comment]
Thanks!  Can you explain exactly what this PR is doing?  I see it adds an 11.3 build, but aside from that, what is the significance of the changes?

**Comment by jaimergp** [comment]
Hi @peastman! Sorry about the noise, the review requests are sent automatically and couldn't avoid the notification. I am just testing a new packaging scheme for `cudatoolkit` which will allow us to use just the needed bits instead of the monolithic package. The details are in the linked issue above. 

No need to worry about this for now. I'll ping you in a comment if needed in future PRs!

**Comment by peastman** [comment]
Ok, thanks!  That sounds useful.

**Pull request closed without merging**